ui/app/templates/components/database-connection.hbs

```diff
@@ -294,14 +291,15 @@
     <div class="field is-grouped is-grouped-split is-fullwidth box is-bottomless">
       <div class="field is-grouped">
         <div class="control">
-          <button data-test-secret-save type="submit" disabled={{this.buttonDisabled}} class="button is-primary">
-            Save
-          </button>
-        </div>
-        <div class="control">
-          <SecretLink @mode="list" class="button">
-            Cancel
-          </SecretLink>
+          <Hds::ButtonSet>
```

Comment:
Manually updated the form buttons to set

<img width="1440" alt="Screenshot 2023-10-17 at 2 08 57 PM" src="https://github.com/hashicorp/vault/assets/82459713/b5148ed0-85a9-42a6-a9df-059530a29855">